frontend/src/app.css

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.home-grid {
+	display: grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	gap: 10px;
+}
```

Comment:
Why not use tailwind for this?

Reply:
They didn't have the right property for grid-template-columns... could have this only be one property I guess?

Reply:
Updated :)